staging/src/k8s.io/apimachinery/pkg/runtime/serializer/protobuf/protobuf.go

```diff
@@ -120,9 +120,7 @@ func (s *Serializer) Decode(originalData []byte, gvk *schema.GroupVersionKind, i
 
 	if intoUnknown, ok := into.(*runtime.Unknown); ok && intoUnknown != nil {
 		*intoUnknown = unk
-		if ok, _, _ := s.RecognizesData(unk.Raw); ok {
-			intoUnknown.ContentType = runtime.ContentTypeProtobuf
-		}
+		intoUnknown.ContentType = runtime.ContentTypeProtobuf
```

Comment:
shouldn't this line be moved up to line 117 so that whatever we do with `unk`, `ContentType` is set correctly?